src/app/core/entity-components/entity-details/entity-details.component.ts

```diff
@@ -37,6 +38,8 @@ export class EntityDetailsComponent {
   entity: Entity;
   creatingNew = false;
 
+  operationType = OperationType;
```

Comment:
always mapping these to a local variable in the components is a pain in the ass ... I am a bit undecided if we shouldn't just use the operation types as plain strings in the templates? (also considering that these might well end up being configurable later)

Reply:
For the moment it gives us type-safetyness and it is also easily traceable where the enums are used. It is easier to change this to just writing strings than coming back from writing strings to this approach.